src/3box.js

```diff
@@ -466,6 +467,7 @@ class Box {
    * @param     {String}        type        The type of link (default 'ethereum')
    */
   async linkAddress (type = ACCOUNT_TYPES.ethereum) {
+    // TODO rebase with addresslink, and will this take proof/args or just default to managementAddress? or another func
```

Comment:
I think it should take proof as an optional param.

Reply:
yeah i think, do mean assuming a proof is a CID?

Reply:
I assume the proof is an object that will be saved into ipfs by this function :)